docs/docsite/rst/user_guide/playbooks_tests.rst

```diff
@@ -61,6 +61,42 @@ To match strings against a substring or a regular expression, use the "match", "
 
 'match' requires zero or more characters at the beginning of the string, while 'search' only requires matching a subset of the string. By default, 'regex' works like `search`, but `regex` can be configured to perform other tests as well.
 
+.. _testing_truthiness:
+
+Testing Truthiness
+``````````````````
+
+.. versionadded:: 2.10
+
+As of Ansible 2.10, you can now perform python like truthy and falsy checks::
```

Comment:
```suggestion
As of Ansible 2.10, you can now perform Python like truthy and falsy checks::
```

Reply:
Please use code-blocks for examples.

How well known is Python truthy/falsy checks?  Is it worth a link to wherever this is discussed in python docs or will most folks already know what it does?  What little I could find calls it truth value.  Which leaves me in a conundrum, because google searches suggest truthy/falsy is what developers are used to calling this (to differentiate it from standard bool true/false) but Python docs themselves don't seem to use it. Leaving to your discretion whether to keep it as truthy/falsy here...